### ISSUE TITLE
Linked Images in card component (used in profile listing) should have null alt attribute (to avoid reading out alt text instead of link destination)

### ISSUE THREAD

**Issue by mmafe** [ccswbs/uofg-components#18]
Linked Images in profile listing should have null alt attribute (to avoid reading out alt text instead of link destination)
<img width="1660" height="893" alt="Image" src="https://github.com/user-attachments/assets/1c4e157b-c6a9-4799-ba3f-389ccf92d4ae" />

https://tq-next.netlify.app/review-profiles-oac
